apps/dispatch-service/src/app/submission/process.processor.ts

```diff
@@ -145,7 +144,8 @@ export class ProcessProcessor extends WorkerHost {
       data: {
         runId: job.data.runId,
         description: 'Post the SED-ML file to the API',
-        moreInfo: undefined,
+        moreInfo: job.data.moreInfo,
+        reason: job.failedReason,
```

Comment:
I cannot find where this field is ever read.  Part of untyped message body, but the message processor for JobQueue.sedmlPost seems to ignore this field.

Does adding this reason field have any effect?

Reply:
I am changing this back to undefined for `moreInfo` for the sake of continuity with the rest of the declarations, but I do think we should eventually implement such fields here so that the post SEDML job and really all the jobs have such metadata. As for `reason`, the same.

Reply:
okay, I understand.  You wanted to put it there towards a better future design.  One thing that bothers me is that the entire data field is defined as type `any`, and we never define types for the payload of each message/job.  In the future, we should define the type of object which goes into the data field (which may be job specific, or not).

approved.